packages/twenty-front/src/modules/ui/field/display/components/PhonesDisplay.tsx

```diff
@@ -30,6 +38,29 @@ const StyledContainer = styled.div`
 `;
 
 export const PhonesDisplay = ({ value, isFocused }: PhonesDisplayProps) => {
+  const { enqueueSnackBar } = useSnackBar();
```

Comment:
let's not call the snackar in such a low component. Instead we should be able to pass call back to the PhoneDisplay component: onPhoneClick and implement the copy behavior (and snackbar) in PhonesFieldMenuItem

Reply:
@charlesBochet Hi charles im not quite sure about implementing the snackbar in phonesFieldMenuItem as i saw the action that if we click on the dropdown then the component is summoned but our main purpose is to copy when its clicked on the chip so im quite not sure how do i implement it there

Reply:
because the PhoneDisplay is too low level (UI) and should not take dependency on the snackbar.
Instead it could expose a prop: onPhoneNumberClick that would allow the parent to decide what to do with this click event (here copy to the copyboard + display snackbar toast)

Reply:
hi @charlesBochet i have edited by passing a prop to phoneDisplay and i have added the functionality to its parent . Can  you review it now?